core/src/main/scala/org/apache/spark/util/RpcUtils.scala

```diff
@@ -54,6 +56,13 @@ private[spark] object RpcUtils {
     RpcTimeout(conf, Seq(RPC_LOOKUP_TIMEOUT.key, NETWORK_TIMEOUT.key), "120s")
   }
 
+  /**
+   * Infinite timeout is used internally, so there's no actual timeout property controls it.
```

Comment:
maybe say "timeout configuration property that controls it".

Reply:
yea, sounds better.